### ISSUE TITLE
Port all command-line tools to cxxopts and remove getopt

### ISSUE THREAD

**Issue by lemire** [simdjson/simdjson#908]
Following PR https://github.com/simdjson/simdjson/pull/904, we have now partially adopted cxxopts in simdjson. However, we still rely on getopt elsewhere. I think we need to finish the job!!!

**Comment by lemire** [comment]
@yoannlr Want to give this a go?

**Comment by lemire** [comment]
Some people suggest that https://github.com/CLIUtils/CLI11 might be a better choice.

At this point in time, I think that cxxopts is progress and while there are better options, it better than getopt.

I suggest we first finish moving to cxxopts and then consider the alternatives. It should be easy to upgrade.

cc @jkeiser

**Comment by lemire** [comment]
Note that cxxopts works with and without exceptions.

**Comment by jkeiser** [comment]
I completely agree that cxxopts is progress, and we should not make the perfect the enemy of the good.

**Comment by lemire** [comment]
@jkeiser There are many things to consider... one of the feature we want to support is the ability to disable exceptions. Cxxopts has that while other frameworks do not document this possibility.

**Comment by AlbertoFVisconti** [comment]
Good morning,
I would like to start contributing to the project and noticed this issue. I know it's 6yo, but I ran grep on the codebase and getopt is still actively used in several command-line tools.
Let me know if this issue is still relevant, thanks.

**Comment by lemire** [comment]
@AlbertoFVisconti Sadly, this issue was resolved some time ago. Closing.
